Tests/Functions/Get-JiraUser.Unit.Tests.ps1

```diff
@@ -138,6 +144,16 @@ Describe "Get-JiraUser" -Tag 'Unit' {
 
             $getResult | Should Not BeNullOrEmpty
 
+            Assert-MockCalled -CommandName Invoke-JiraMethod -Exactly 1 -Scope It -ParameterFilter {$URI -like "$jiraServer/rest/api/*/user/search?*username=$testUsername*"}
+            Assert-MockCalled -CommandName Invoke-JiraMethod -Exactly 1 -Scope It -ParameterFilter {$URI -like "$jiraServer/rest/api/*/user?username=$testUsername&expand=groups"}
+        }
+
+        It "Gets information about a provided Jira exact user" {
+            $getResult = Get-JiraUser -UserName $testUsername
+
+            $getResult | Should Not BeNullOrEmpty
+
+            Assert-MockCalled -CommandName Invoke-JiraMethod -Exactly 1 -Scope It -ParameterFilter {$Method -eq 'Get' -and $URI -like "$jiraServer/rest/api/*/user?username=$testUsername"}
```

Comment:
`-Exactly 0` ?

Reply:
No, -Exactly 1. To limit the changes and potential for things going wrong I haven't changed that behaviour. The `Get-JiraUser` command will still get the user as it does when searching, and in both cases it will go then to `user?username=$testUsername&expand=groups"`. If it was to be a permanent, long term solution it could be optimised, but that would mean more changes and more risk that something gets broken.